themes/bootstrap3/templates/_ui/components/menu-button.phtml

```diff
@@ -0,0 +1,46 @@
+<?php
+  $wrapperTag = $this->wrapperTag ?? 'div';
+  $wrapperClass = 'dropdown'; // Bootstrap class
+
+  if ($this->wrapperClass ?? false) {
+    $wrapperClass = $this->wrapperClass . ' dropdown'; // Bootstrap class
+  }
+
+  $toggleTag = 'button';
+  $toggleAttrs = $this->toggleAttrs ?? [];
+  $toggleAttrs['type'] = 'button'; // Accessible default
+  $toggleAttrs['class'] = 'dropdown-toggle'; // Bootstrap class
+  $toggleAttrs['data-toggle'] = 'dropdown'; // Bootstrap data-attr
+  $toggleAttrs['aria-expanded'] = 'false';
+  $toggleAttrs['aria-haspopup'] = 'true';
+
+  // An anchor tag with an href should only be used as
+  // a non-js fallback to open the menu on another page
+  if ($this->href ?? false) {
+    $toggleTag = 'a';
+    $toggleAttrs['href'] = $this->href;
+    $toggleAttrs['type'] = null;
+  }
+?>
+
+<?php ob_start(); ?>
+  <ul class="dropdown-menu">
+    <?php foreach ($this->menuItems as $current): ?>
+      <li class="dropdown__item <?php if ($current['selected'] ?? false): ?> active<?php endif ?>">
+        <a class="dropdown__link" href="<?=$this->escapeHtmlAttr($current['url']) ?>" rel="nofollow">
+          <?=$this->transEsc($current['label']) ?>
+        </a>
+      </li>
+    <?php endforeach; ?>
+  </ul>
+
+  <?=$this->makeTag(
+    $toggleTag,
+    $this->transEsc($this->toggleLabel) . $this->icon('dropdown-caret'),
+    $toggleAttrs,
+    ['escapeContent' => false]
+) ?>
```

Comment:
Should this line be indented to align with 37?

Reply:
I agree. `php-cs-fixer` seems to like this because it seems to mix 2 and 4 space indenting.

Reply:
Yeah, I see that `php-cs-fixer` is still being a pain. Maybe something like this would satisfy it while being more clear?

```
  <?=$this
      ->makeTag(
        $toggleTag,
        $this->transEsc($this->toggleLabel) . $this->icon('dropdown-caret'),
        $toggleAttrs,
        ['escapeContent' => false]
      )
  ?>
```

(I haven't tried it, so maybe that won't work any better... let me know if you need me to put more time into fighting this!

Reply:
I'm happy with their counter to this offer:

```php
  <?=$this
      ->makeTag(
          $toggleTag,
          $this->transEsc($this->toggleLabel) . $this->icon('dropdown-caret'),
          $toggleAttrs,
          ['escapeContent' => false]
      )
  ?>
```